src/main/java/Duke.java

```diff
@@ -1,10 +1,171 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
There are few or no [Javadoc comments](https://se-education.org/guides/conventions/java/basic.html#comments). Consider supplying method header comments for longer methods to improve code readability.